### PR TITLE
Adds the Liquid Drama reagent.

### PR DESCRIPTION
**Pull request by Iamgoofball**
Adds the liquid drama reagent and 5 extra reagents for the recipe.

### CODE REVIEW

**Comment by Iamgoofball** [comment]
FUCKING WANT TO RIP YOUR GUTS OUT AND STRANGLE YOU WITH THEM RIGHT NOW @THEVEKTER

File: code/modules/reagents/Chemistry-Goon-Other.dm [hunk 1]
```diff
@@ -284,4 +284,19 @@ datum/reagent/concentrated_barbers_aid/reaction_mob(var/mob/living/M, var/volume
...
+	id = "liquid_drama"
+	description = "All the hatred of the galaxy, compacted into 1 liquid."
+	reagent_state = LIQUID
+	color = "#C8A5DC" // rgb: 200, 165, 220
+
+/datum/chemical_reaction/liquid_drama
+	name = "liquid_drama"
+	id = "liquid_drama"
+	result = "liquid_drama"
+	required_reagents = list("stardust" = 1, "eye_of_toad" = 1,  "solid_errors" = 1, "liquid_rage" = 1, "paprika" = 1)
```
Comment (Anonus) [inline code/modules/reagents/Chemistry-Goon-Other.dm]: No cat hair? Shame.

**Comment by TheVekter** [comment]
@Iamgoofball was trying to ninja #8000 by being lame and merging an empty PR, for shame :-1:

File: code/modules/reagents/Chemistry-Goon-Other.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/modules/reagents/Chemistry-Goon-Other.dm]: no cattebeest

**Comment by TZK13** [comment]
>No singulo in the list of required reagents.

It's even a real reagent in the code.

File: code/modules/reagents/Chemistry-Goon-Other.dm (hunk 1 above)
Comment (TZK13) [inline code/modules/reagents/Chemistry-Goon-Other.dm]: >No singulo.

It's even a real reagent.

Comment (Iamgoofball) [inline code/modules/reagents/Chemistry-Goon-Other.dm]: i could add singulo yeah

will do

**Comment by dannno** [comment]
both of you fucked up

**Comment by JJRcop** [comment]
I had [this](https://github.com/JJRcop/-tg-station/commit/e334830cd0099b28dc59abf74d97aed53f23e326) prepared for #8000, but I was playing vidya and didn't keep track of the issue numbers...

**Comment by paprka** [comment]
>eating paprika doesn't give you toxin damage or brain damage

goof how do you manage to fuck up even a simple joke

**Comment by palpatine213** [comment]
Please actually make this a thing. This sounds amazing.

**Comment by Cheridan** [comment]
Don't you think that this is in poor taste?

**Comment by paprka** [comment]
That's why it gives you burn damage though cheridan

:+1: 

I think he really just wanted to make a joke 8000 PR like ayyran does

**Comment by palpatine213** [comment]
Make an adminverb to inject liquid drama into players. It might be a fun thing for admins to do if a shitstorm is happening somewhere.

File: code/modules/reagents/Chemistry-Goon-Toxins.dm [hunk 2]
```diff
@@ -204,4 +204,18 @@ datum/reagent/itching_powder/on_mob_life(var/mob/living/M as mob)
...
+	name = "Paprika"
+	id = "paprika"
+	description = "A spice made from chili peppers."
+	reagent_state = LIQUID
+	color = "#CF3600" // rgb: 207, 54, 0
+
+datum/reagent/paprika/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	if(prob(50))
+		M.adjustFireLoss(1)
```
Comment (phil235) [inline code/modules/reagents/Chemistry-Goon-Toxins.dm]: Please no, don't add fire damage to it if you add it to chili pepper, we don't need more harmful recipes. Plus, capsaicin already exists.

Comment (RemieRichards) [inline code/modules/reagents/Chemistry-Goon-Toxins.dm]: Add fire STACKS instead of fire damage.
that's more fun.

**Comment by phil235** [comment]
Adding paprika (with fire damage or firestacks) and including it into chili peppers would be a death sentence to all chili pepper based recipes. :-1:

**Comment by Chocobro** [comment]
just have paprika plants. blend them to get salt

**Comment by Iamgoofball** [comment]
I'll move paprika to a special plant mutation

**Comment by paprka** [comment]
this isn't even #8000 

the joke is ded

**Comment by Jordie0608** [comment]
http://keepachangelog.com/

**Comment by Dorsisdwarf** [comment]
Player references aren't allowed and you missed your joke spot.

**Comment by Cheridan** [comment]
le gimmicks

**Pull request closed without merging**